static/app/views/performance/transactionSummary/transactionAnomalies/content.tsx

```diff
@@ -174,24 +186,17 @@ type DataType = {
   chart: AnomalyData;
 };
 
-function Anomalies(props: Props) {
+function Anomalies(props: AnomaliesSectionProps) {
   const height = 250;
   const chartColor = theme.charts.colors[0];
 
   const chart = useMemo<QueryDefinition<DataType, WidgetDataResult>>(() => {
     return {
       fields: '',
-      component: provided => (
-        <AnomaliesQuery
-          orgSlug={props.organization.slug}
-          location={props.location}
-          eventView={props.eventView}
-          {...pick(provided, 'children')}
-        />
-      ),
+      component: provided => <Fragment>{provided.children(props.queryData)}</Fragment>,
```

Comment:
This is a workaround to keep within the bounds of the existing`GenericPerformanceWidget` implementation, without resorting to discriminated unions of props or any of the more technically correct but equally messy solutions.

Long term if we add hook queries then this becomes trivial to remove (since all of the `queries` for GenericPerformanceWidget can be simplified to hooks).